sdk/src/main/java/com/relaypro/sdk/Relay.java

```diff
@@ -29,10 +29,7 @@
 import java.net.http.HttpResponse;
 import java.time.Duration;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
This may be more of a Relay specific convention, but I do prefer to import specifically-named classes instead of using wildcards, so it is more clear what you have imported when looking at class names in the source.